crates/dbs-interrupt/src/notifier.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2019 Alibaba Cloud. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+//! Device event notifier to inject interrupt to guest.
+
+use std::io::Error;
+use std::sync::Arc;
+
+use downcast_rs::{impl_downcast, Downcast};
+use vmm_sys_util::eventfd::EventFd;
+
+use crate::{InterruptIndex, InterruptSourceGroup, InterruptStatusRegister32};
+
+#[cfg(feature = "legacy-irq")]
+pub use self::legacy::*;
```

Comment:
why using "pub use" here?

Reply:
Re-exporting the `LegacyNotifier` and `MsiNotifier` struct.